Jantimizer/QueryTestSuite/Parsers/MySQLParser.cs

```diff
@@ -25,7 +25,13 @@ public override AnalysisResult ParsePlan(DataSet planData)
                 throw new NoNullAllowedException($"Unexpected null-value from MySql Analysis");
             }
 
-            return RunAnalysis(resultQueue);
+            return Task.FromResult(RunAnalysis(resultQueue));
```

Comment:
Why not just straight up return the result instead? The `RunAnalysis(...)` function is not `async` anyway, so you gain nothing from making it a task.